build.sbt

```diff
@@ -58,6 +57,6 @@ startYear := Some(2016)
 organizationHomepage := Some(url("https://github.com/sangria-graphql"))
 developers := Developer("OlegIlyenko", "Oleg Ilyenko", "", url("https://github.com/OlegIlyenko")) :: Nil
 scmInfo := Some(ScmInfo(
-  browseUrl = url("https://github.com/sangria-graphql/sangria-msgpack.git"),
-  connection = "scm:git:git@github.com:sangria-graphql/sangria-msgpack.git"
-))
\ No newline at end of file
+  browseUrl = url("https://github.com/sangria-graphql-org/sangria-msgpack.git"),
+  connection = "scm:git:git@github.com:sangria-graphql-org/sangria-msgpack.git"
+))
```

Comment:
I guess we'll also need:
```
releaseCrossBuild := true
releasePublishArtifactsAction := PgpKeys.publishSigned.value
```

Reply:
Thanks, good catch—trying to do these between other things and missed that.